hw2/script-1.js

```diff
@@ -0,0 +1,18 @@
+let myName = "Mariia";
+let mySurmame = "Nazarova";
+let myCity = "Bilgorod-Dnistrovskyi";
+let a = (2 + 3 + 4 + 6) / 4;
```

Comment:
As for me better to use more meaningful names for variables, it's okay to name radius as `r` but `a` have not much sense.
Also, some of the variables, such as name or radius can be initialized as constants, because values will be unchanged